samples/bot-request-approval/nodejs/.env

```diff
@@ -1,3 +1,5 @@
 MicrosoftAppId=
 MicrosoftAppPassword=
-BaseUrl=
\ No newline at end of file
+BaseUrl=
+MicrosoftAppType=
```

Comment:
Remove extra app setrtings and update the same in localyml file

Reply:
fixed